packages/fetchai/connections/p2p_libp2p/connection.py

```diff
@@ -705,7 +705,7 @@ async def _receive_from_node(self) -> None:
             assert self._in_queue is not None, "Input queue not initialized."
             self._in_queue.put_nowait(data)
             if data is None:
-                break
+                break  # pragma: no cover
```

Comment:
on my local machine, this is already covered by  `tests/test_packages/test_connections/test_p2p_libp2p/test_errors.py::TestP2PLibp2pConnectionFailureNodeDisconnect::test_node_disconnect`
(https://github.com/fetchai/agents-aea/blob/develop/tests/test_packages/test_connections/test_p2p_libp2p/test_errors.py#L143)